server/api/jurisdictions.py

```diff
@@ -805,6 +826,28 @@ def get_batch_comparison_audit_discrepancies_by_jurisdiction(
 
             audited_contest_result = audited_batch_result[contest.id]
             reported_contest_result = reported_batch_results[batch_key][contest.id]
+
+            # Special case: for combined batches, only count discrepancies in the representative batch
+            combined_batch_name = None
+            if batch_key in all_combined_batch_keys:
+                if batch_key not in representative_batch_to_combined_batch:
+                    continue
+                combined_batch = representative_batch_to_combined_batch[batch_key]
+                combined_batch_name = combined_batch["name"]
+                sub_batches = combined_batch["sub_batches"]
+                sub_batch_reported_results = list(
+                    sub_batch.jurisdiction.batch_tallies[sub_batch.name].get(contest.id)  # type: ignore
+                    for sub_batch in sub_batches
+                )
+                reported_contest_result = {
+                    choice.id: sum(
+                        reported_results[choice.id]
+                        for reported_results in sub_batch_reported_results
+                        if reported_results is not None
+                    )
+                    for choice in contest.choices
+                }
```

Comment:
After reading this code, I went back and checked out how we compute the discrepancy counts for combined batches, and I think there's a bug there, since we only use the reported results from the representative batch rather than summing all the sub-batches. The test didn't catch it because the test data still resulted in a discrepancy, just not the correct one.

I think we should remove that endpoint anyways and just count the discrepancies sent to the frontend, so it's probably moot, but that's a motivating reason to do that refactor sooner rather than later.

Reply:
Hm right, good catch. I'm off tomorrow as well, but happy to take it on early next week if we'd like

Reply:
I'll just knock it out right now. Should be quick

Reply:
https://github.com/votingworks/arlo/pull/2051